File: Daily Test/C++/Toggle - Digits & Words 26.10.2025.cpp
#include <iostream>
#include <cstring>
#include <cctype>
using namespace std;

int main() {
    string L[] = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
    string s;
    cin >> s;
    int i = 0, len = s.length();
    while (i < len) {
        if (isdigit(s[i])) {
            cout << s[i];
            i++;
        } else {
            string p = "";
            while (i < len && isalpha(s[i])) {
                p += s[i++];
            }
            for (int j = 0; j < 10; j++) {
                if (p == L[j]) {
                    cout << j;
                    break;
                }
            }
        }
    }
    return 0;
}

File: Daily Test/Python/Toggle - Digits & Words 26.10.2025.py
s = input().strip()
L = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
i = 0
while i < len(s):
    if s[i].isdigit():
        print(s[i], end="")
        i += 1
    else:
        p = ''
        while i < len(s) and s[i].isalpha():
            p += s[i]
            i += 1
        for j in range(len(L)):
            if p == L[j]:
                print(j, end="")
                break